<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LT-019-PC" name="Default test" tests="2" failures="0" timestamp="12 Jul 2013 12:03:53 GMT" time="0.033" errors="0">
  <testcase name="@BeforeClass beforeSuite" time="0.0" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <failure type="com.etouch.taf.exception.InvalidTestProfileException" message="Config file cannot be null">
      <![CDATA[com.etouch.taf.exception.InvalidTestProfileException: Config file cannot be null
	at com.etouch.taf.core.ProfileConfiguration.<init>(ProfileConfiguration.java:40)
	at com.etouch.taf.core.TestBed.<init>(TestBed.java:17)
	at com.etouch.taf.TestBedManager.<init>(TestBedManager.java:18)
	at com.etouch.taf.core.BaseTest.beforeSuite(BaseTest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeSuite -->
  <testcase name="@BeforeClass loadTestData" time="0.0" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <skipped/>
  </testcase> <!-- @BeforeClass loadTestData -->
  <testcase name="@BeforeClass prepareBeforeClass" time="0.0" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <skipped/>
  </testcase> <!-- @BeforeClass prepareBeforeClass -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.etouch.taf.core.BaseTest.tearDown(BaseTest.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:225)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:114)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="testValidateHeader" time="0.003" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <skipped/>
  </testcase> <!-- testValidateHeader -->
  <testcase name="testValidateHeader1" time="0.001" classname="test.etouch.foxpoc.shakenbake.TestShakeAndBakeBlog">
    <skipped/>
  </testcase> <!-- testValidateHeader1 -->
</testsuite> <!-- Default test -->
